website/content/docs/v0.10/Guides/disaster-recovery.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Disaster Recovery"
+description: "Procedure for snapshotting etcd database and recovering from catastrophic control plane failure."
+---
+
+`etcd` database backs Kubernetes control plane state, so if `etcd` service is unavailable
+Kubernetes control plane goes down, and the cluster is not recoverable until `etcd` is recovered with contents.
+`etcd` consistency model builds around consensus protocol Raft, so for highly-available control plane clusters,
+loss of one control plane doesn't impact cluster health.
+In general, `etcd` stays up as long as quorum nodes are up.
+For three control plane node Talos cluster, this means that the cluster tolerates a failure of any single node,
+but losing more than one node leads to complete cluster failure.
+Because of that, it is important to take backups of `etcd` state to have a snapshot to recover cluster from
+in case of catastrophic failure.
+
+## Backup
+
+### Snapshotting `etcd` Database
+
+Create a consistent snapshot of `etcd` database with `talosctl etcd snapshot` command:
+
+```bash
+$ talosctl -n <IP> etcd snapshot db.snapshot
+etcd snapshot saved to "db.snapshot" (2015264 bytes)
+snapshot info: hash c25fd181, revision 4193, total keys 1287, total size 3035136
+```
+
+This database snapshot can be taken on any healthy control plane node, as all `etcd` instances contain
+exactly same data.
+Configure `etcd` snapshots to be created on some schedule to allow point-in-time recovery using the latest snapshot.
+
+### Disaster Database Snapshot
+
+If `etcd` cluster is not healthy command `talosctl etcd snapshot` might fail.
+In that case, as a last resort, copy the database snapshot from the control plane node:
+
+```bash
+talosctl -n <IP> cp /var/lib/etcd/member/snap/db .
+```
+
+This snapshot might not be fully consistent, but it allows for disaster recovery when last consistent
```

Comment:
Snapshots pulled this way are, in fact, guaranteed to be consistent (unless I am very mistaken), but they do not (apparently) include the check hash.

Reply:
yep, thank you!

Reply:
but I guess if you copy it while `etcd` is running, it won't be consistent, as there's no way writes are synchronized with copy.

Reply:
I rephrased this sentence a bit.